client/src/reducers/centroidLabels.ts

```diff
@@ -1,20 +1,16 @@
-import type { Action } from "redux";
+import type { AnyAction } from "redux";
 
 export interface CentroidLabelsState {
   showLabels: boolean;
 }
 
-export interface CentroidLabelsAction extends Action<string> {
-  showLabels: boolean;
-}
-
 const initialState: CentroidLabelsState = {
   showLabels: false,
 };
 
 const centroidLabels = (
   state = initialState,
-  action: CentroidLabelsAction,
```

Comment:
Since `CentroidLabelsAction` is used in this reducer, let's keep it instead of using `AnyAction`! Thank you!

Here and throughout